pontoon/base/tests/test_models.py

```diff
@@ -1542,41 +1542,85 @@ def test_has_suggestions_plural(self):
         TranslationFactory.create(
             locale=self.plural_locale,
             entity=first_entity,
-            approved=True,
+            approved=False,
             fuzzy=False,
             plural_form=1,
         )
         TranslationFactory.create(
             locale=self.plural_locale,
-            entity=first_entity,
+            entity=third_entity,
             approved=False,
             fuzzy=False,
-            plural_form=2,
+            plural_form=0,
         )
         TranslationFactory.create(
             locale=self.plural_locale,
             entity=third_entity,
             approved=True,
             fuzzy=False,
-            plural_form=0,
+            plural_form=1
```

Comment:
Let's keep trailing comas, they make subsequent diffs easier to read if we're ever to add a new argument to this function.